OpenIPSL/UsersGuide/GettingStarted.mo

```diff
@@ -23,13 +23,12 @@ simulation tool, e.g., <a href=\"http://faraday1.ucd.ie/psat.html\">PSAT</a> and
 You can use these tools to create a power flow solution for your network.
 If you do not have access to these tools or do not want to use them, there are several power flow solvers
 available on Github.
-Future work in the <strong><code>OpenIPSL</code></strong> effort will include to generate Modelica records from open source power flow solvers
-such as <a href=\"https://github.com/SanPen/GridCal\">GridCal</a> or <a href=\"https://github.com/FRESNA/PyPSA\">PyPSA</a>.
+Examples on how to generate Modelica records from open source and proprietary power flow solvers, specifically <a href=\"https://github.com/SanPen/GridCal\">GridCal</a> and <a href=\"https://new.siemens.com/global/en/products/energy/energy-automation-and-smart-grid/pss-software/pss-e.html\">PSS&reg;E</a> can be found in <a href=\"https://ecp.ep.liu.se/index.php/modelica/article/view/190/150\">this paper</a> and in <a href=\"https://github.com/ALSETLab/SMIB_Tutorial\">this Github repository</a>.
 </p>
 <p>
 From these values, a Modelica tool solves the initialization problem for all algebraic and differential - state variables.
-All models in <strong><code>OpenIPSL</code></strong> are programmed in such way that by introducing a <a href=\"https://github.com/SmarTS-Lab/Raw2Record\">power flow solution</a> (from another tool), the <strong>initial guesses</strong> are computed as parameters within each model and are provided into the initial equations that are used to solve the overall initialization problem.
-See <a href=\"http://www.ep.liu.se/ecp/article.asp?issue=119&amp;article=010\">this paper</a> for a more detailed explanation.
+All models in <strong><code>OpenIPSL</code></strong> are programmed in such way that by introducing a <a href=\"https://github.com/ALSETLab/SMIB_Tutorial\">power flow solution</a> (from another tool), the <strong>initial guesses</strong> are computed as parameters within each model and are provided into the initial equations that are used as an initial guess to solve the overall initialization problem.
+See <a href=\"https://ecp.ep.liu.se/index.php/modelica/article/view/190/150\">this paper</a> for a more detailed explanation.
```

Comment:
Probably better to use the DOI here.

Reply:
Yes, good idea!
BTW, don't you have paper's that should be listed too? @dietmarw